src/event/handlers/LogMemberLeaveHandler.ts

```diff
@@ -0,0 +1,27 @@
+import { Constants, GuildMember, MessageEmbed, TextChannel } from "discord.js";
+import EventHandler from "../../abstracts/EventHandler";
+import { EMBED_COLOURS, MEMBER_ROLE, LOG_CHANNEL_ID } from "../../config.json";
+import DateUtils from "../../utils/DateUtils";
+
+class LogMemberLeaveHandler extends EventHandler {
+	constructor() {
+		super(Constants.Events.GUILD_MEMBER_REMOVE);
+	}
+
+	async handle(guildMember: GuildMember): Promise <void> {
+		const embed = new MessageEmbed();
+
+		embed.setTitle(`${guildMember.user.tag} left the server.`);
+		embed.setColor(EMBED_COLOURS.DEFAULT);
+		embed.addField("Join date", new Date(guildMember.joinedTimestamp!).toLocaleString(), true);
+		embed.addField("Leave date", new Date(Date.now()).toLocaleString(), true);
+		embed.addField("Time in server", DateUtils.getFormattedTimeSinceDate(guildMember.joinedTimestamp!, Date.now()));
```

Comment:
Each word in the title of these fields should begin with a capital letter as it is a heading.